packages/cli/src/commands/oracle/report.ts

```diff
@@ -0,0 +1,66 @@
+import { CeloContract, CeloToken } from '@celo/contractkit'
+import { flags } from '@oclif/command'
+import BigNumber from 'bignumber.js'
+import { BaseCommand } from '../../base'
+import { displaySendTx, failWith } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+
+export default class ReportPrice extends BaseCommand {
+  static description =
+    'Report the price of Celo Gold in a specified token (currently just Celo Dollar, aka: "StableToken")'
+
+  static flags = {
+    ...BaseCommand.flags,
+    from: Flags.address({ required: true, description: 'Address of the oracle account' }),
+    token: flags.string({
+      required: true,
+      description: 'The token to report on',
+    }),
+    numerator: flags.string({
+      required: true,
+      description: 'Amount of the specified token equal to the amount of cGLD in the denominator',
+    }),
+    denominator: flags.string({
+      required: false,
+      description: 'Amount of cGLD equal to the numerator. Defaults to 1 if left blank',
+    }),
+  }
+
+  static example = [
+    'report --token StableToken --numerator 1.02 --from 0x8c349AAc7065a35B7166f2659d6C35D75A3893C1',
+    'report --token StableToken --numerator 102 --denominator 100 --from 0x8c349AAc7065a35B7166f2659d6C35D75A3893C1',
+  ]
+
+  async run() {
+    const res = this.parse(ReportPrice)
+    let token: CeloToken
+    if (res.flags.token === CeloContract.StableToken) {
+      token = CeloContract.StableToken
```

Comment:
Does token have to be set if `res.flags.token` is equal to StableToken? Alternatively, you could set the `options` option on the flag to require token to be member of a set